tokenManager.js

```diff
@@ -1,15 +1,15 @@
-require("dotenv").config();
-const secretKey = process.env.SECRET_KEY;
+import "dotenv/config";
 import JWT from "jsonwebtoken";
+const secretKey = process.env.SECRET_KEY;
 let loggedInUsers = [];
```

Comment:
What happens if `SECRET_KEY` *isn't* in `.env` (even though it's supposed to be)?